src/features/surveys/components/SurveySubmissionsList.tsx

```diff
@@ -20,6 +14,12 @@ import { usePanes } from 'utils/panes';
 import ZUIPersonGridCell from 'zui/ZUIPersonGridCell';
 import ZUIPersonGridEditCell from 'zui/ZUIPersonGridEditCell';
 import ZUIRelativeTime from 'zui/ZUIRelativeTime';
+import {
+  DataGridPro,
+  GridCellParams,
+  GridRenderCellParams,
+  useGridApiContext,
+} from '@mui/x-data-grid-pro';
```

Comment:
The only thing that has happened in this file is that these imports moved from the first block of imports (where they should be) to the second block of imports. Be careful in the future not to commit changes that have no meaning. For this, I use `git add -p` to inspect every change before adding it to my commit, but you can do the same in VSCode by looking at the diff before committing.

Reply:
Thanks for the tip!  🙏